css/style.css

```diff
@@ -680,15 +705,34 @@ button.button-inline:hover {
 
 .cell-board-title {
 	width: 50%;
+}
 
+#boardlist tr.deleted td * {
```

Comment:
It will also change the opacity of the border that separates the lines, that looked a bit odd. :wink: